trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk

```diff
@@ -10,31 +11,76 @@
 {% endblock %}
 
 {% set premiumPaymentsHtml %}
-  {% if errorSummary %}
+  {% if premiumPaymentFormError or facilityIdQueryError %}
     {{ govukErrorSummary({
       titleText: "There is a problem",
-      errorList: errorSummary,
+      errorList: [premiumPaymentFormError, facilityIdQueryError],
```

Comment:
Does nothing strange happen when one of these errors is `undefined`?

Reply:
It doesn't seem to but I would thought something weird might happen also:
<img width="2093" alt="Screenshot 2024-07-03 at 11 27 11" src="https://github.com/UK-Export-Finance/dtfs2/assets/67907735/b9d10491-d568-45ac-8f6f-7b153bbfffda">